plone/namedfile/scaling.py

```diff
@@ -17,7 +17,12 @@
 from zope.traversing.interfaces import ITraversable, TraversalError
 from zope.publisher.interfaces import IPublishTraverse, NotFound
 from zope.app.file.file import FileChunk
-from plone.protect.interfaces import IDisableCSRFProtection
+
+try:
+    # Soft dependency to make this package work without plone.protect
+    from plone.protect.interfaces import IDisableCSRFProtection
+except ImportError:
```

Comment:
There was a discussion elsewhere - using pkg_resources is slower than just using ImportError. It does only mask errors in the package, from which we are importing, which should catched via tests anyways. Aslo depending on pkg_resources, which comes from setuptools does not feel right within production code (IMO).
Where was this discussion .... can't really remember...